packages/eslint-plugin-react-hooks/src/ExhaustiveDeps.js

```diff
@@ -1653,7 +1653,7 @@ function analyzePropertyChain(node, optionalChains) {
   }
 }
 
-function getNodeWithoutReactNamespace(node, options) {
+function getNodeWithoutReactNamespace(node) {
```

Comment:
This is actually the only little thing I would agree in this PR, there should be not unused params indeed.